.github/workflows/build.yml

```diff
@@ -49,34 +54,45 @@ jobs:
       run: |
         cd Brogue.app/Contents && dylibbundler -cd -b -x MacOS/brogue
 
+    - name: "Create artifact"
+      run: |
+        make BrogueCE-macos
+        zip -rll BrogueCE-macos-x86_64.zip BrogueCE-macos
+
     - name: "Upload artifact"
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@v2
       with:
-        name: Brogue.app
-        path: Brogue.app
+        name: macos-x86_64
+        path: BrogueCE-macos-x86_64.zip
 
   windows:
     runs-on: windows-latest
     steps:
     - name: "Checkout sources"
       uses: actions/checkout@v2
+
     - name: "Install dependencies"
       run: |
         .\.github\get-deps-mingw.ps1
+
     - name: "Compile"
       shell: powershell
       run: |
         $env:path = $env:path + ";" + (Join-Path $pwd.Drive.Root "opt/local/x86_64-w64-mingw32/bin")
         $env:path = $env:path + ';C:\Program Files (x86)\Windows Kits\10\bin\x64'
         mingw32-make CC=gcc bin/brogue.exe
-    - name: "Package"
+
+    - name: "Create artifact"
       run: |
-        cp .\SDL2\x86_64-w64-mingw32\bin\SDL2.dll .\bin\
-        cp .\SDL2_image\x86_64-w64-mingw32\bin\zlib1.dll .\bin\
-        cp .\SDL2_image\x86_64-w64-mingw32\bin\SDL2_image.dll .\bin\
-        cp .\SDL2_image\x86_64-w64-mingw32\bin\libpng16-16.dll .\bin\
+        mingw32-make BrogueCE-windows
+        cp SDL2\x86_64-w64-mingw32\bin\SDL2.dll BrogueCE-windows
```

Comment:
I'd still push these extra steps back down into the build.  Same with the macos `dylibbundler` step above.

Reply:
dylibbundler maybe, but disagree for this Windows stuff. This has a very particular dependency on the specific CI build environment and the steps preceding it. It makes no sense IMO to move this into the release Make rule which may be done in a completely different environment. I think it would be the wrong level of abstraction.

Reply:
Going to target this for a future update to BUILD.md explaining more about steps for releases